packages/core/src/Layout/LayoutManager.ts

```diff
@@ -81,6 +85,6 @@ class LayoutManager {
   }
 }
 
-module.exports = {
+export {
   LayoutManager,
 };
```

Comment:
Nit: the export could be `export class LayoutManager` to keep consistency with Layout